### PR TITLE
Adding ScreenshotsUtils

### PR DESCRIPTION
**Pull request by EngyAAshraf**

### CODE REVIEW

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        Path path = Paths.get("test-outputs/screenshots",screenshotname+".png");
+        try {
+            Files.createDirectories(path.getParent());
+            FileOutputStream out = new FileOutputStream(path.toString());
+            out.write(((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES));
+            out.close();
+        }
+        catch (Exception e) {
+            System.out.println("Exception while take screenshot"+e.getMessage());
```
Comment (Ashraaf7) [inline src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:23]: Keep in mind that we will replace this printing line with a Loggin

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ScreenshotsUtils
+{
+    public static  void capturescreenshot(WebDriver driver, String screenshotname) {
+
+        Path path = Paths.get("test-outputs/screenshots",screenshotname+".png");
```
Comment (Ashraaf7) [inline src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:15]: Also, here we will concatenate the TimeStamp after the screenshot name keep this in mind.

**Review by Ashraaf7 — CHANGES_REQUESTED** [review]

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (EngyAAshraf) [reply src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:15]: private why?

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ScreenshotsUtils
+{
+    public static  void capturescreenshot(WebDriver driver, String screenshotname) {
```
[the remark this answers is not in the document]
Comment (EngyAAshraf) [reply src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:13]: Do you mean that ?
    public static  Files capturescreenshot(WebDriver driver, String screenshotname) {

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java (hunk 1 above)
Comment (EngyAAshraf) [reply src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:23]: okay

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java (hunk 2 above)
Comment (EngyAAshraf) [reply src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:15]: Now or do you mean later?

**Review by Ashraaf7 — CHANGES_REQUESTED** [review]

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
+import java.io.File;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
 public class ScreenshotsUtils
 {
     private static Path path ;
-
+    private static String screenshotPath = "test-outputs/screenshots";
```
Comment (Ashraaf7) [inline src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:13]: should be CONSTANT.

File: src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java [hunk 5]
```diff
@@ -25,50 +20,66 @@ public void setPath(Path path)
...
             LogUtils.error(e.getMessage());
             return null;
         }
     }
-    //create method to Take screenshot of specified WebElement only
-    public static FileOutputStream captureWebElementScreenshots(WebDriver driver, WebElement element) {
-
-        path = Paths.get("test-outputs/screenshots",element+".png");
-        try {
+    //create method to Take screenshot with high lighting element
```
Comment (Ashraaf7) [inline src/main/java/com/automationexercise/Utilities/ScreenshotsUtils.java:62]: highlighting*

**Review by Ashraaf7 — CHANGES_REQUESTED** [review]

**Review by Ashraaf7 — APPROVED** [review]

**Pull request merged**